.github/workflows/package.yaml

```diff
@@ -8,6 +8,7 @@ on:
 jobs:
   build:
     runs-on: ${{ matrix.config.os }}
+    environment: packaging
```

Comment:
Spravanie tu je ake? "Dedi" to ine envars alebo pouziva len envarss z toho environemntu?

Reply:
dedi sa to s tym, najnizsi level overriduje

> If a secret with the same name exists at multiple levels, the secret at the lowest level takes precedence. For example, if an organization-level secret has the same name as a repository-level secret, then the repository-level secret takes precedence. Similarly, if an organization, repository, and environment all have a secret with the same name, the environment-level secret takes precedence.